.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main, typescript ]
+  pull_request:
+    branches: [ main, typescript ]
+  schedule:
```

Comment:
Why this is called by schedule?

Reply:
It's from a template provided directly by GitHub/CodeQL. The reason is to have new scan results even if there is no activity on the repository.

It's optional. I can drop it if you don't like it.

Reply:
I don't think that this is needed.

Reply:
I have dropped `on: schedule` trigger.